tests/integration/nmstatectl_test.py

```diff
@@ -35,6 +36,7 @@
 SHOW_CMD = ["nmstatectl", "show"]
 CONFIRM_CMD = ["nmstatectl", "commit"]
 ROLLBACK_CMD = ["nmstatectl", "rollback"]
+VERSION_CMD = ["nmstatectl", "version"]
```

Comment:
You do not need to use a constant for the version command because it is used only once. Please use it as a tuple directly in the function call. The other commands are extended in the tests, therefore they need to be lists.